README.md

```diff
@@ -132,11 +133,25 @@ The `fugue` API consists of four functions:
 -   `fugue.missing_resource(resource_type)` marks a resource as **missing**.
     This is useful if you for example _require_ a log group to be present.
 
-## Control mapping
+### Rule library
```

Comment:
Should we note which ones are proper rules and which are examples, so people know which ones to pick and choose?

Reply:
Also, should we add links to the files?

Reply:
I actually didn't include any of the examples here. Perhaps we add another subsection on examples?

Reply:
Oh right -- sorry! That would be nice. Or at least a sentence mentioning the folder and why the rules there aren't included with the rest. The directory structure is listed further down so we don't need to go into much detail, I think.